src/Mapbender/CoreBundle/Resources/public/mapbender.geosource.js

```diff
@@ -243,12 +246,12 @@ Mapbender.Geo.SourceHandler = Class({
                     if (layer.children[i].options.id.toString() === offsetLayer.options.id.toString()) {
                         options.found = true;
                         if (options.cut_with) {
```

Comment:
Variable names and object keys should be always lover camel case

Reply:
Code not touched, same as before, avoiding mixing in too much refactoring with a behaviour fix.

Reply:
... buuuuut renamed to `ìncludeOffset` now. That's the name of the value in the object passed in to getLayersList. Kind of makes sense to keep it constant through the call chain, and it's more intuitive this way what it's doing.